subprojects/logging/src/main/java/org/gradle/internal/logging/events/RenderableOutputEvent.java

```diff
@@ -16,13 +16,16 @@
 
 package org.gradle.internal.logging.events;
 
+import org.gradle.api.Nullable;
 import org.gradle.api.logging.LogLevel;
 import org.gradle.internal.logging.text.StyledTextOutput;
 
 public abstract class RenderableOutputEvent extends CategorisedOutputEvent {
+    private final OperationIdentifier operationId;
```

Comment:
Great comment @bmuschko. I see more `OperationIdentifier` on `OutputEvent` as a stop gap while we migrate toward `BuildOperation`. With `BuildOperation`, I think we should be using `BuildOperationDetails#getOperationDescriptor()` to get that kind of information. @jjohannes Would that be the right statement to do here?